app/src/main/java/org/breezyweather/sources/CommonConverter.kt

```diff
@@ -550,6 +552,117 @@ private fun computeWetBulbTemperature(
         - 4.686035)
 }
 
+/**
+ * Compute mean sea level pressure (MSLP) from barometric pressure and altitude.
+ * Optional elements can be provided for minor adjustments.
+ * Source: https://integritext.net/DrKFS/correctiontosealevel.htm
+ *
+ * To compute barometric pressure from MSLP,
+ * simply enter negative altitude.
+ *
+ * @param barometricPressure in hPa
+ * @param altitude in meters
+ * @param temperature in °C (optional)
+ * @param humidity in % (optional)
+ * @param latitude in ° (optional)
+ */
+fun computeMeanSeaLevelPressure(
+    barometricPressure: Double?,
+    altitude: Double?,
+    temperature: Double? = null,
+    humidity: Double? = null,
+    latitude: Double? = null,
+): Double? {
+    // There is nothing to calculate if barometric pressure or altitude is null.
+    // Return the barometric pressure as it is.
+    if (barometricPressure == null || altitude == null) {
+        return barometricPressure
```

Comment:
It should return `null`, to avoid any risk of mixing of absolute and sea-level pressure in charts

Reply:
Fixed.